### ISSUE TITLE
TypeScript errors with version 7

### ISSUE THREAD

**Issue by boris-petrov** [qonto/ember-autofocus-modifier#522]
Thanks for the new release!

When using the Glint types that come with the plugin, I get the following errors:

```
app/components/something.hbs:205:63 - error TS2345: Argument of type 'HTMLDivElement' is not assignable to parameter of type 'PrebindArgs<HTMLElement, "normalize">'.
  Property '[NamedArgs]' is missing in type 'HTMLDivElement' but required in type 'NamedArgsMarker'.

205           <div {{(if this.modalShown (modifier "autofocus" "button"))}}>
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/.pnpm/@glint+template@1.2.2/node_modules/@glint/template/-private/integration.d.ts:69:3
    69   [NamedArgs]: true;
         ~~~~~~~~~~~
    '[NamedArgs]' is declared here.

app/components/something.hbs:205:107 - error TS2769: No overload matches this call.
  The last overload gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'NamedArgs<{ disabled?: boolean | undefined; }>'.
      Type 'string' is not assignable to type 'NamedArgsMarker'.

205           <div {{(if this.modalShown (modifier "autofocus" "button"))}}>
                                                               ~~~~~~~~

  node_modules/.pnpm/@glint+template@1.2.2/node_modules/@glint/template/-private/keywords/-bind-invokable.d.ts:80:3
     80   <
          ~
     81     Positional extends any[],
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    ...
     93     ) => Return
        ~~~~~~~~~~~~~~~
     94   >;
        ~~~~
    The last overload is declared here.
```

Any ideas? I was using very similar definition until now that works fine:

```ts
declare class AutofocusModifier extends Modifier<{
  readonly Args: {
    readonly Positional: [string?];
    readonly Named: { readonly disabled?: boolean | undefined };
  };
}> {}

...

autofocus: typeof AutofocusModifier;
```

**Comment by dannycalleri** [comment]
Hey @boris-petrov , thanks for opening this.

Did I understand correctly that you are using `@glint/template@1.2.2`?
Could you provide the full code snippet?

Thanks!

**Comment by boris-petrov** [comment]
@dannycalleri thanks for the response!

Yes, I'm using Glint 1.2.2. As for a code snippet - this reproduces the issue in isolation:

```hbs
<div {{(if true (modifier "autofocus" "button"))}}>
</div>
```

The error I see in VSCode with the Glint extension or by running `./node_modules/.bin/glint` in the command line.

Do you have troubles reproducing the problem?

**Comment by vscav** [comment]
Hey @boris-petrov 👋  We have a potential fix in review right now - https://github.com/qonto/ember-autofocus-modifier/pull/529 (I managed to reproduce using your snippet, thanks!)

**Comment by vscav** [comment]
@boris-petrov The fix is available in the last release ([v7.0.1](https://github.com/qonto/ember-autofocus-modifier/releases/tag/v7.0.1)). Let us know if there is anything else 😊
